mkdocs.yml

```diff
@@ -68,6 +68,7 @@ nav:
           - api/layer-extensions/brushing-extension.md
           - api/layer-extensions/collision-filter-extension.md
           - api/layer-extensions/data-filter-extension.md
+          - api/layer-extensions/path-style-extension.md
```

Comment:
This file doesn't get auto-generated. We have to create a new markdown at that location. You can look at `data-filter-extension.md` for the syntax.